src/components/players/details/index.module.scss

```diff
@@ -14,7 +14,8 @@
   justify-content: center;
   align-items: flex-end;
   height: 292px;
-  background-image: url('../../../assets/imgs/player-backdrop-large.png');
+  background-image: url('../../../assets/imgs/player-backdrop.png');
+  background-size: cover;
```

Comment:
All the code in here (aside from `.container`) is not being used. Remove unused scss code